common/djangoapps/student/management/tests/test_bulk_unenroll.py

```diff
@@ -114,20 +98,19 @@ def test_users_unenroll_successfully_logged(self):
             else:
                 users_unenrolled[username] = [str(enrollment.course.id).encode('utf-8')]
 
-            lines += str(enrollment.user.id) + "," + username + "," + \
-                enrollment.user.email + "," + str(enrollment.course.id) + "\n"
+            lines += username + "," + str(enrollment.course.id) + "\n"
 
         csv_file = SimpleUploadedFile(name='test.csv', content=lines.encode('utf-8'), content_type='text/csv')
         BulkUnenrollConfiguration.objects.create(enabled=True, csv_file=csv_file)
 
         with LogCapture(LOGGER_NAME) as log:
             call_command("bulk_unenroll")
+            for key, value in users_unenrolled.items():
+                username, course = key, value[0].decode('utf-8')
             log.check(
```

Comment:
I believe you need to indent this into the loop block, otherwise it will only verify the log for the last item in that list.